libavformat/mxfdec.c

```diff
@@ -883,6 +959,30 @@ static int mxf_read_strong_ref_array(AVIOContext *pb, UID **refs, int *count)
     return 0;
 }
 
+static inline int mxf_read_string(AVIOContext *pb, int size, char** str)
```

Comment:
I would call this `mxf_read_us_ascii_string` since MXF strings can also be `UTF16`.